src/radium-theme.js

```diff
@@ -30,7 +30,8 @@ module.exports = {
   // Media Queries
   // --------
   mediaSizes: {
-    medium: "only screen and (min-width: 44em)"
+    medium: "only screen and (min-width: 44.063em)",
+    small: "only screen and (max-width: 44em)"
```

Comment:
yeah, you're right. I should spend more time refactoring the existing MQs instead of just adding an inverse breakpoint. I'll take care of this!